### PR TITLE
feat: update code examples to use Angular 15

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

File: projects/docs-tools-showcase/src/app/home/mock-documentation.json [hunk 1]
```diff
@@ -10,29 +10,27 @@
     "SkyColorpickerMessage": "interface-skycolorpickermessage",
     "SkyColorpickerOutput": "interface-skycolorpickeroutput",
     "SkyColorpickerResult": "interface-skycolorpickerresult",
-    "SkyColorpickerRgba": "interface-skycolorpickerrgba",
-    "SkyColorpickerUtils": "class-skycolorpickerutils"
+    "SkyColorpickerRgba": "interface-skycolorpickerrgba"
```
Comment (Blackbaud-SteveBrush) [inline projects/docs-tools-showcase/src/app/home/mock-documentation.json:13]: Were these changes intentional?

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Review by Blackbaud-SteveBrush — CHANGES_REQUESTED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

File: projects/docs-tools-showcase/src/app/home/mock-documentation.json (hunk 1 above)
Comment (johnhwhite) [reply projects/docs-tools-showcase/src/app/home/mock-documentation.json:13]: It reflects the new dev-infra setup, not necessarily related to code-examples, but is related to Sky UX 8.
